bugbot/rules/workflow/no_severity_nag.py

```diff
@@ -0,0 +1,199 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from datetime import datetime
+
+import numpy
+from libmozdata import utils as lmdutils
+
+from bugbot import utils
+from bugbot.bzcleaner import BzCleaner
+from bugbot.escalation import Escalation
+from bugbot.nag_me import Nag
+from bugbot.round_robin import RoundRobin
+
+
+class NoSeverityNag(BzCleaner, Nag):
+    def __init__(self, inactivity_days: int = 4):
+        """Constructor
+
+        Args:
+            inactivity_days: number of days that a bug should be inactive before
+                being considered.
+        """
+        super(NoSeverityNag, self).__init__()
+        self.lookup_second = utils.get_config(self.name(), "weeks_lookup", 4)
```

Comment:
No need to call this "second" anymore

Reply:
Fixed here: 416ddf650a8a0970454f774bd76e533bcbbdcc2e